### PR TITLE
Future AnimalGenomics talks

### PR DESCRIPTION
**Pull request by bjlang**
@netlify /events/2024/SIG_animalgenomics_June

### CODE REVIEW

File: src/content/events/2024/SIG_animalgenomics_June [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: '#animal-genomics: Inaugural Meeting'
```
Comment (JoseEspinosa) [inline src/content/events/2024/SIG_animalgenomics_June:2]:
```suggestion
title: '#animal-genomics: TBC'
```

File: src/content/events/2024/SIG_animalgenomics_May [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: '#animal-genomics: Inaugural Meeting'
```
Comment (JoseEspinosa) [inline src/content/events/2024/SIG_animalgenomics_May:2]:
```suggestion
title: '#animal-genomics: TBC'
```

File: src/content/special-interest-groups/animal-genomics/meetings.md [hunk 3]
```diff
@@ -11,9 +11,17 @@ of every month at 4 PM CET / 10 AM EDT / 7 AM PDT for 1h.
 
 The first meeting will be on Wednesday, April 17th, at 16:00 CEST. For this inaugural session, we are honored to announce Christa Kühn, president of the Friedrich Loeffler Institute, BovReg project coordinator and former coordinator of EuroFAANG RI, as the speaker. Christa is well-known in the Bovine field for her important contributions to animal genetics. Her talk will be followed by a round table where the structure of the series will be announced and discussed with the participants.
```
Comment (JoseEspinosa) [inline src/content/special-interest-groups/animal-genomics/meetings.md:12]: Maybe we should update this then to be a general introductory text for all the meetings?

File: src/content/special-interest-groups/animal-genomics/meetings.md [hunk 4]
```diff
@@ -11,9 +11,17 @@ of every month at 4 PM CET / 10 AM EDT / 7 AM PDT for 1h.
...
 
+| Date   |      Speaker       |
+|----------:|:-------------|
+| Apr 17 | Christa Kühn (BovReg; Friedrich Loeffler Institute, Germany) |
```
Comment (JoseEspinosa) [inline src/content/special-interest-groups/animal-genomics/meetings.md]: Could we add the link to the respective event here?

**Review by JoseEspinosa — CHANGES_REQUESTED** [review]

**Comment by bjlang** [comment]
@nf-core-bot fix linting

**Comment by bjlang** [comment]
@nf-core-bot fix linting

**Review by JoseEspinosa — APPROVED** [review]
Awesome job! 🚀

**Pull request merged**